src/storage/backend/sqlite/SQLite.cc

```diff
@@ -237,12 +237,18 @@ OperationResult SQLite::DoErase(ResultCallback* cb, ValPtr key) {
 void SQLite::DoExpire(double current_network_time) {
     auto stmt = expire_stmt.get();
 
-    if ( auto res = CheckError(sqlite3_bind_double(stmt, 1, current_network_time)); res.code != ReturnCode::SUCCESS ) {
-        sqlite3_reset(stmt);
-        // TODO: do something with the error here?
+    int status = sqlite3_bind_double(stmt, 1, current_network_time);
+    if ( status != SQLITE_OK ) {
+        // TODO: do something with the error?
```

Comment:
I guess? This shouldn't happen and we run at a rather low frequency IIUC, so maybe `fprintf(stderr, ...` at least?

Mind putting something in the commit message what the data race actually was? Something in `Check()` or `Step()` ?